tests/test_components/test_loader.py

```diff
@@ -101,9 +108,10 @@ def test_component_loading_module_not_found_error_framework_package_with_wrong_a
     ):
         with pytest.raises(
             AEAPackageLoadingError,
-            match="No module named packages.some_author; No AEA package found with author name 'some_author'",
-        ):
+            match="An error occurred while loading protocol an_author/a_protocol:0.1.0:",
+        ) as e:
             load_component_from_config(component_configuration)
+            assert "some_type" in str(e)
```

Comment:
Please check for a more complete error message (like on LHS); so we see in the tests what is expected.

Reply:
it's the second check, just to see the most informative part of the specific exception

Reply:
I know, but for test writers and reviewers (us) the previous bigger output was more helpful when reading tests.

Reply:
Just have a longer string there so we can see what the full error message says (excluding the trace)